extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1709,6 +1709,77 @@ describes.realWin(
           'Swipe up'
         );
       });
+
+      describe('remote locales', () => {
```

Comment:
Here and in the main code file, please be consistent with your vocabulary. locales/localizationStrings etc, please pick one and stick to it for clarity.